.github/workflows/build_and_publish.yml

```diff
@@ -0,0 +1,75 @@
+name: Buld and Publish
```

Comment:
Why did change this process? Could you instead keep the same pypi-release.yml file as before (which comes from https://github.com/aboutcode-org/skeleton/blob/main/.github/workflows/pypi-release.yml and should be update there) so we can see the diff of changes you propose? Also make this a separate PR.